src/components/Slider/Slider.jsx

```diff
@@ -0,0 +1,242 @@
+
+const Slider = forwardRef(
+  (
+    {
+      ariaLabel,
+      ariaLabeledBy,
+      className,
+      classNameBase,
+      color,
+      "data-testid": dataTestId,
+      disabled,
+      id,
+      isRange,
+      max,
+      min,
+      onChange,
+      step,
+      showValue,
+      size,
+      value,
+      valueDefault,
+      valueFormatter,
+      valueText,
+      // ------ Additional subcomponents' props
+      indicateSelection,
+      prefix,
+      postfix
+    },
+    ref
+  ) => {
+    const componentRef = useRef(null);
+    const mergedRef = useMergeRefs({ refs: [ref, componentRef] });
+    const consumerBem = createBemBlockHelper(classNameBase, { isConsume: true });
+    const providerProps = {
+      ariaLabel,
+      ariaLabeledBy,
+      classNameBase,
+      color,
+      dataTestId,
+      disabled,
+      isRange,
+      max,
+      min,
+      onChange,
+      showValue,
+      size,
+      step,
+      value,
+      valueDefault: ensureValueDefault({ valueDefault, isRange, min, max }),
+      valueFormatter,
+      valueText,
+      infixOptions: {
+        prefix,
+        postfix,
+        indicateSelection
+      }
+    };
+    return (
+      <SliderProvider {...providerProps}>
+        <div
+          className={bem("", { disabled, "value-shown": showValue }, className)}
+          data-testid={dataTestId}
+          id={id}
+          ref={mergedRef}
+        >
+          <SliderInfix kind="prefix" />
+          <SliderBase className={consumerBem("base")} />
+          <SliderInfix kind="postfix" />
+        </div>
+      </SliderProvider>
+    );
+  }
+);
+
+Slider.sizes = SIZES_BASIC;
+Slider.colors = COLORS_BASIC;
+
+Slider.propTypes = {
+  // ------ SliderBase props
+  /**
+   * Define a string that labels the current element (Slider)
+   */
+  ariaLabel: PropTypes.string,
+  /**
+   * ElementId of Node that have the text needed for labeling the current element (Slider)
+   */
+  ariaLabeledBy: PropTypes.string,
+  /**
+   * Custom `class name` to be added to the component-root-node
+   */
+  className: PropTypes.string,
+  /**
+   * Custom base of class names for add to component-root and internal-components nodes
+   *  - check Dev Tools for found available Custom/Consumer Classes
+   */
+  classNameBase: PropTypes.string,
+  /**
+   * Color Mode (primary/positive/negative) of the component (Slider)
+   */
+  color: PropTypes.oneOf(Object.values(Slider.colors)),
+  /**
+   * Unique TestId - can be used as Selector for integration tests and other needs (tracking, etc)
+   */
+  "data-testid": PropTypes.string,
+  /**
+   * If set to true, Component (Slider) will be disabled
+   *  - impossible to change value of component (Slider)
+   *  - visual changes (opacity)
+   */
+  disabled: PropTypes.bool,
+  /**
+   * Attribute `id` to be added to the component-root-node
+   */
+  id: PropTypes.string,
+  /**
+   * If true switch slider to RRange mode (two Thumbs)
+   */
+  isRange: PropTypes.bool,
```

Comment:
we don't use anymore on is prefix. I think we can remove this prop ad calculate if the slide is in range value according to it's value, if the value is array with 2 value therefore we will display a range type slider.

Reply:
We can change `isRange` to `ranged` (by analogy with `disabled`)

First of all `value` is pretty rare use case for controlled version of slider.
So probably we are speaking about defaultValue

But an Idea was  exactly wise versa - we just providing `ranged` props and getting defaultValue=[min, max] or  (exactly as min for non-ranged)

In other case we suddenly have to provide pretty complex explanation why defaultValue switching mode.
And users need to provide defaultValue=[min, max] - even id they didn't need it. 

ranged - looks more self explaining and simple for most use-cases.

Reply:
let's change the prop name to ranged